MiniApp/Classes/Extensions/MiniApp+Date.swift

```diff
@@ -0,0 +1,6 @@
+extension Date {
+    var currentDate: String {
```

Comment:
I would give this a more explicit name, like `epochInMillisecondString` or `epochInMillisecond`

Reply:
Done